### PR TITLE
Fixes #28 Update B2D to v1.13.1

### LINKED ISSUE #28: Update the ISO to docker v1.13.1
At this point we are running relatively old version of docker i.e. 1.12.3 and we should update it latest docker i.e. https://github.com/boot2docker/boot2docker/releases/tag/v1.13.1

### PR DESCRIPTION
**Pull request by gbraad**
Fixes #28

Artifacts:
  * https://gitlab.com/gbraad/minishift-b2d-iso/-/jobs/29206639/artifacts/file/public/minishift-b2d.iso
  * http://artifacts.ci.centos.org/minishift/minishift-b2d-iso/pr/54/

### CODE REVIEW

File: iso/Dockerfile [hunk 1]
```diff
@@ -14,15 +14,15 @@
 
 # TODO: Move this to somewhere on GCR and manage tags explicitly instead of
 # relying on latest.
-FROM boot2docker/boot2docker:1.12.3
+FROM boot2docker/boot2docker:1.13.1
 
 # Set the version of Docker and the expected sha.
-RUN echo "1.12.3" > $ROOTFS/etc/version
+RUN echo "1.13.1" > $ROOTFS/etc/version
 ENV DOCKER_SHA a3bb88649bcc47775c29dab7d95709f4dda04d0e
```
Comment (brgnepal) [inline iso/Dockerfile]: @gbraad You need to update `DOCKER_SHA` as well

**Comment by gbraad** [comment]
Already did so on my personal copy on GitLab. Was away from home...
Build passed already ;-) https://gitlab.com/gbraad/minishift-b2d-iso/-/jobs/29206639

Note: wasn't even aware I already proposed it as a PR.

**Comment by brgnepal** [comment]
With this PR, another similar PR https://github.com/minishift/minishift-b2d-iso/pull/50 looks invalid.

WDYT @gbraad @hferentschik @LalatenduMohanty @praveenkumar  ?

Can we close #50 ?

**Comment by praveenkumar** [comment]
@gbraad @budhrg I am fine with this PR as long as we consist about our docker version across the ISO as I mentioned to #50 also.

**Comment by hferentschik** [comment]
Was the plan not to have aligned versions of Docker in B2D and CentOS ISO? I seem to recall that we said we will defer upgrading to the latest available version and hence deferring issue #28. Instead we said we align versions. Do I recall this wrong? Have we changed the opinion?

**Comment by hferentschik** [comment]
@praveenkumar @gbraad @budhrg - seems we forgot about this and pr #50. Can we get either merge #50 or this modify this pull request in order to upgrade the Docker version _and_ align with what we have in CentOS ISO. After that we should be able to cut a release and make sure to update to the latest version in Minishift.

**Comment by brgnepal** [comment]
> this modify this pull request in order to upgrade the Docker version and align with what we have in CentOS ISO

#50 align the docker version with that of centos iso. I suggest to pull changes from #50 and add your changes on top of that as additional commit.

**Comment by hferentschik** [comment]
> I suggest to pull changes from #50 and add your changes on top of that as additional commit.

+1 Whatever makes sense. Would be nice to get this merged asap, so that we have a full sprint to use the new updated ISO.

**Pull request closed without merging**